### PR TITLE
Update Community page with just one code review required for a PR

### PR DESCRIPTION
**Pull request by leventov**
Discussion is here: https://groups.google.com/forum/#!msg/druid-development/CB8Fc68oko4/z8cQVw-HAwAJ

### CODE REVIEW

**Comment by himanshug** [comment]
:+1:

**Comment by cheddar** [comment]
:+1:

File: community/index.md [hunk 1]
```diff
@@ -26,19 +26,28 @@ into, or adding features you need, are both immensely helpful.
...
-exception is major architectural changes or API changes, which often warrant
-additional review. For those, it can help to discuss things on the [druid-development
-list](https://groups.google.com/d/forum/druid-development) or a github issue beforehand.
+Patches to Druid are done through GitHub pull requests.
+
+Pull requests require one approval (+1) from an established committer on code and text (for documentation) levels. The
+exception is major architectural changes or API changes, and/or changes to
+
+ - HTTP requests and responses (e. g. a new HTTP endpoint)
+ - Interfaces for extensions
```
Comment (leventov) [inline community/index.md]: Added

File: community/index.md [hunk 2]
```diff
@@ -26,19 +26,28 @@ into, or adding features you need, are both immensely helpful.
...
+ - Server configuration (e. g. altering the behavior of a config property)
+ - Emitted metrics
```
Comment (leventov) [inline community/index.md]: Added

**Comment by fjy** [comment]
👍

**Comment by gianm** [comment]
Looks like this has enough votes, shall we merge and announce the new rules on druid-development?

**Pull request merged**